Add github light (#49)
Quansight-Labs/jupyterlab-accessible-themes: packages/githublight/README.md

> # Github Light

Reviewer: ```suggestion
# GitHub Light
```

> A theme that mimics github light theme. It is heavily inspired in the `github light theme default` for VSCode available [here](https://github.com/primer/github-vscode-theme).

Reviewer: ```suggestion
A theme that mimics GitHub's light theme. It is heavily inspired by the [`GitHub Light Default` theme for VSCode](https://github.com/primer/github-vscode-theme).
```

Reviewer 2: This seems unnecessarily redundant:

```suggestion
A theme heavily inspired by the [`GitHub light default theme` for VSCode](https://github.com/primer/github-vscode-theme).
```

## Usage ✨

> To enable the theme in JupyterLab: `Go to Settings -> JupyterLab Theme -> Github Light`

Reviewer: ```suggestion
You enable the theme from the JupyterLab menu bar. Go to `Settings -> JupyterLab Theme -> Github Light`
```

## Colors

> All the colors used for text in the theme are compliant with [WCAG 2.1 standards](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

Reviewer: ```suggestion
All the colors used for text in the theme are compliant with [WCAG 2.1 contrast standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
```

Reviewer 2: Is it AA/AAA - we should clarify here.
Also, what about non-text elements? Are these also WCAG-conformant?

Also let's use conformant over compliant as we are adhering to the standards rather than being certified or similar

Reviewer 2: just saw the table below so perhaps indicating we have a table here would be useful

Author: > Also, what about non-text elements? Are these also WCAG-conformant?

These are harder to test and present here just because there's a lot of possible combinations around the UI. I can try to add them just to see how many there are.

Reviewer 2: We do not need to add this to a table. 
But since we are saying that the themes are conformant we do need to make sure that this the case. Either through manual or automatic testing.

I think having the table was useful for the syntax highlighting themes as the colour combination are smaller but having it in this theme makes it slightly redundant. Will defer to @gabalafou for further discussion too

Background color: ![##ffffff](https://via.placeholder.com/20/#ffffff/#ffffff.png) `#ffffff`

Highlight color: ![#321f7ad9](https://via.placeholder.com/20/321f7ad9/321f7ad9.png) `#321f7ad9`

> **WCAG compliance**

Reviewer: I'm not sure how to read this table. The contrast values in the table are for each color against a white (#fff) background, correct? 

Should the last two columns reversed? I would have thought that the second row, for the color #6e7781, would be AA for normal text, but AAA for large.

Reviewer 2: Agree - the table or some text should specify which colour combinations are being displayed (white/colours or highlight/colours).

+1 on the columns seeming reversed

| ![#57606a](https://via.placeholder.com/20/57606a/57606a.png) | `#57606a` | 6.4 : 1  | AAA         | AA         |
| ![#424a53](https://via.placeholder.com/20/424a53/424a53.png) | `#424a53` | 9.0 : 1  | AAA         | AAA        |
| ![#32383f](https://via.placeholder.com/20/32383f/32383f.png) | `#32383f` | 11.8 : 1 | AAA         | AAA        |
| ![#24292f](https://via.placeholder.com/20/24292f/24292f.png) | `#24292f` | 14.7 : 1 | AAA         | AAA        |
| ![#0550ae](https://via.placeholder.com/20/0550ae/0550ae.png) | `#0550ae` | 7.6 : 1  | AAA         | AAA        |
| ![#0a3069](https://via.placeholder.com/20/0a3069/0a3069.png) | `#0a3069` | 12.8 : 1 | AAA         | AAA        |
| ![#116329](https://via.placeholder.com/20/116329/116329.png) | `#116329` | 7.4 : 1  | AAA         | AAA        |
| ![#953800](https://via.placeholder.com/20/953800/953800.png) | `#953800` | 7.4 : 1  | AAA         | AAA        |
| ![#a40e26](https://via.placeholder.com/20/a40e26/a40e26.png) | `#a40e26` | 7.9 : 1  | AAA         | AAA        |
| ![#6639ba](https://via.placeholder.com/20/6639ba/6639ba.png) | `#6639ba` | 7.3 : 1  | AAA         | AAA        |

## Screenshots

> <img alt="JupyterLab Github Light screenshot" src="./images/screenshot.png"/>

Reviewer: ```suggestion
<img alt="JupyterLab GitHub Light screenshot" src="./images/screenshot.png"/>
```

## Acknowledgements

> This theme is based and inspired in the [`github light default` theme for VSCode](https://github.com/primer/github-vscode-theme)

Reviewer: ```suggestion
This theme is based on and inspired by the [`GitHub Light Default` theme for VSCode](https://github.com/primer/github-vscode-theme)
```

| ![#57606a](https://via.placeholder.com/20/57606a/57606a.png) | `#57606a` | 6.4 : 1  | AA          | AAA        |
| ![#424a53](https://via.placeholder.com/20/424a53/424a53.png) | `#424a53` | 9.0 : 1  | AAA         | AAA        |
| ![#32383f](https://via.placeholder.com/20/32383f/32383f.png) | `#32383f` | 11.8 : 1 | AAA         | AAA        |
| ![#24292f](https://via.placeholder.com/20/24292f/24292f.png) | `#24292f` | 14.7 : 1 | AAA         | AAA        |
| ![#0550ae](https://via.placeholder.com/20/0550ae/0550ae.png) | `#0550ae` | 7.6 : 1  | AAA         | AAA        |
| ![#0a3069](https://via.placeholder.com/20/0a3069/0a3069.png) | `#0a3069` | 12.8 : 1 | AAA         | AAA        |
| ![#116329](https://via.placeholder.com/20/116329/116329.png) | `#116329` | 7.4 : 1  | AAA         | AAA        |
| ![#953800](https://via.placeholder.com/20/953800/953800.png) | `#953800` | 7.4 : 1  | AAA         | AAA        |
| ![#a40e26](https://via.placeholder.com/20/a40e26/a40e26.png) | `#a40e26` | 7.9 : 1  | AAA         | AAA        |
| ![#6639ba](https://via.placeholder.com/20/6639ba/6639ba.png) | `#6639ba` | 7.3 : 1  | AAA         | AAA        |

## Font

> This theme is using the [Atkinson Hyperlegible font](https://fonts.google.com/specimen/Atkinson+Hyperlegible), which was developed specifically to increase legibility for readers with low vision.

Reviewer 2: Some observations:

1. If you are planning to use this typeface for all the themes then you also need to update it in pitaya smoothie to unify the behaviour
2. We should be linking to the Atkinson Hyperlegible website and not google fonts https://brailleinstitute.org/freefont
3. Also we should use the description the typeface uses itself - AFAIK the typeface was designed for readability rather than being better suited for low-vision folks.

# GitHub Light

A theme heavily inspired by the [`GitHub light default theme` for VSCode](https://github.com/primer/github-vscode-theme).

## Usage ✨

To enable the theme from the JupyterLab menu bar. Go to `Settings -> JupyterLab Theme -> Github Light`

## Colors

All the colors used for text in the theme are conformant with [WCAG 2.1 contrast standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).

Background color: ![##ffffff](https://via.placeholder.com/20/#ffffff/#ffffff.png) `#ffffff`

Highlight color: ![#321f7ad9](https://via.placeholder.com/20/321f7ad9/321f7ad9.png) `#321f7ad9`

> **WCAG compliance**

Reviewer 2: ```suggestion
**WCAG conformance**
```
